src/main/java/io/jenkins/plugins/opentelemetry/backend/grafana/LokiLogStorageRetriever.java

```diff
@@ -127,7 +125,7 @@ public LokiLogStorageRetriever(@Nonnull String lokiUrl, boolean disableSslVerifi
 
     @Nonnull
     @Override
-    public LogsQueryResult overallLog(@Nonnull String jobFullName, int runNumber, @Nonnull String traceId, @Nonnull String spanId, boolean complete, @Nonnull Instant startTime, @Nullable Instant endTime) {
+    public LogsQueryResult overallLog(String jobFullName, int runNumber, String traceId, String spanId, boolean complete, Instant startTime, @Nullable Instant endTime) {
```

Comment:
@Nonnull is not defined in the parent method definition, so it does not have an effectt